norduser/service/fork.go

```diff
@@ -71,14 +71,19 @@ func getRunningNorduserPIDs() ([]int, error) {
 }
 
 func findPIDOfUID(uids string, desiredUID uint32) int {
-	for _, pidUid := range strings.Split(uids, "\n") {
-		var pid, uid int
-		_, err := fmt.Sscanf(pidUid, " %d %d", &uid, &pid)
+	for _, uidPid := range strings.Split(uids, "\n") {
+		var pid int
+		var uid int
+		n, err := fmt.Sscanf(uidPid, "%d%d", &uid, &pid)
```

Comment:
Is the format string `"%d%d"` correct?

Reply:
Yes, the spaces are discarded, so it should work with combinations of variable number of spaces and pids/uids. Confirmed with uts.